module/VuFind/src/VuFind/ILS/Logic/RenewalsHelper.php

```diff
@@ -77,20 +79,22 @@ public function addRenewDetails($catalog, $ilsDetails, $renewStatus)
     /**
      * Process renewal requests.
      *
-     * @param \Laminas\Stdlib\Parameters $request       Request object
-     * @param \VuFind\ILS\Connection     $catalog       ILS connection object
-     * @param array                      $patron        Current logged in patron
-     * @param CsrfInterface              $csrfValidator CSRF validator
+     * @param Parameters         $request        Request object
+     * @param Connection         $catalog        ILS connection object
+     * @param array              $patron         Current logged in patron
+     * @param FlashMessenger     $flashMessenger Flash messenger for user messages
+     * @param CsrfInterface|null $csrfValidator  CSRF validator
```

Comment:
The other syntax is also correct, but this is more concise (and more commonly used in the codebase):

```suggestion
     * @param Parameters     $request        Request object
     * @param Connection     $catalog        ILS connection object
     * @param array          $patron         Current logged in patron
     * @param FlashMessenger $flashMessenger Flash messenger for user messages
     * @param ?CsrfInterface $csrfValidator  CSRF validator
```